tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py

```diff
@@ -32,7 +30,7 @@ def tap_cover_in_player_view(self):
             Wait(self.marionette).until(expected.element_present(*self._rating_view_locator))))
 
     def _get_star_locator(self, rating):
-        return (By.CSS_SELECTOR, '.rating-star[data-rating="%s"]' % rating)
+        return By.CSS_SELECTOR, '.rating-star[data-rating="%s"]' % rating
```

Comment:
I still think this is requierd. We need to make sure we return a tuple from this method.
https://github.com/chirarobert/gaia/blob/bug_1119278/tests/python/gaia-ui-tests/gaiatest/apps/music/regions/player_view.py#L41